### ISSUE TITLE
detecting fusion

### ISSUE THREAD

**Issue by Aseem139** [royerlab/ultrack#126]
Is this tool capable of detecting objects that fuse over time as mitochondria do ?

**Comment by JoOkuma** [comment]
@Aseem139 , no, we don't allow fusion. You could process it backward, but them you will won't detect divisions.
